boards/silabs/radio_boards/xg28_rb4401c/doc/index.rst

```diff
@@ -0,0 +1,138 @@
+.. zephyr:board:: xg28_rb4401c
+
+Overview
+********
+
+The EFR32ZG28 Radio Board is the radio board delivered with
+`_xG28-PK6024A Website`_. It contains a Wireless System-On-Chip from the
+EFR32ZG28 family built on an ARM Cortex®-M33 processor with excellent low
+power capabilities.
+
+The BRD4401C a.k.a. xG28-RB4401C radio board plugs into the Wireless Pro Kit
+Mainboard BRD4401C and is supported as one of :ref:`silabs_radio_boards`.
+
+Hardware
+********
+
+- EFR32ZG28B322F1024IM68 SoC
+- CPU core: ARM Cortex®-M33 with FPU
+- Flash memory: 1024 kB
+- RAM: 256 kB
+- Transmit power: up to +20 dBm
+- Operation frequency: 868-915 MHz
+- Crystals for LFXO (32.768 kHz) and HFXO (39 MHz).
+- Silicon Labs Si7021 relative humidity and temperature sensor
+- Low-power 128x128 pixel Memory LCD
+- Macronix ultra low power 8-Mbit SPI flash (MX25R8035F)
+
+For more information about the EFR32ZG28 SoC and BRD4401C board, refer to these
+documents:
+
+- `EFR32ZG28 Website`_
+- `EFR32ZG28 Datasheet`_
+- `EFR32xG28 Reference Manual`_
+- `XG28-PK6024A Website`_
+- `BRD4401C User Guide`_
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Connections and IOs
+===================
+
+In the following table, the column **Name** contains Pin names. For example, PA2
+means Pin number 2 on PORTA, as used in the board's datasheets and manuals.
+
++-------+-------------+-------------------------------------+
+| Name  | Function    | Usage                               |
++=======+=============+=====================================+
+| PD11  | UART_TX     | UART Console TX                     |
++-------+-------------+-------------------------------------+
+| PD12  | UART_RX     | UART Console RX                     |
++-------+-------------+-------------------------------------+
+| PB1   | GPIO        | Push Button 0                       |
++-------+-------------+-------------------------------------+
+| PB2   | GPIO        | LED0                                |
++-------+-------------+-------------------------------------+
+| PB3   | GPIO        | Push Button 1                       |
++-------+-------------+-------------------------------------+
+| PD3   | GPIO        | LED1                                |
++-------+-------------+-------------------------------------+
+| PD10  | SPI_CS      | SPI chip select                     |
```

Comment:
This is incorrect and also doesn't appear in your DTS. Please add the display, flash and temperature/humidity sensor to the device tree and fix the doc